onnxruntime/test/fuzzing/src/test.cpp

```diff
@@ -180,61 +165,52 @@ int processCommandLine(int argc, char* argv[], runtimeOpt& opt)
       char desiredScale;
       parser << argv[index--];
       parser >> desiredScale;
-      
-      if (parser.bad())
-      {
+
+      if (parser.bad()) {
         throw std::exception("Could not parse the time scale from the command line");
       }
-      
+
       opt.scale = static_cast<timeScale>(std::tolower(desiredScale));
-      switch(opt.scale)
-      {
+      switch (opt.scale) {
         case timeScale::Hrs:
         case timeScale::Min:
         case timeScale::Sec:
           break;
         default:
           throw std::exception("Could not parse the time scale from the command line");
       }
-      
+
       parser << argv[index--];
       parser >> opt.testTimeOut;
-      if (parser.bad())
-      {
+      if (parser.bad()) {
         throw std::exception("Could not parse the time value from the command line");
       }
 
-      Logger::testLog<< L"Running Test for: " << opt.testTimeOut << desiredScale << Logger::endl;
+      Logger::testLog << L"Running Test for: " << opt.testTimeOut << desiredScale << Logger::endl;
       opt.modelFileName = Logger::towstr(argv[index--]);
-      Logger::testLog<< L"Model file: " << opt.modelFileName << Logger::endl;
+      Logger::testLog << L"Model file: " << opt.modelFileName << Logger::endl;
       std::filesystem::path modelFileNamePath{opt.modelFileName};
-      if (!std::filesystem::exists(modelFileNamePath))
-      {
+      if (!std::filesystem::exists(modelFileNamePath)) {
         throw std::exception("Cannot find model file");
       }
 
       // process options
       //
-      while(index > 0)
-      {
+      while (index > 0) {
         auto option{std::string{argv[index]}};
-        if ( option == "/m")
-        {
+        if (option == "/m") {
           opt.userOpt.writeModel = true;
-        }
-        else if (option == "/v")
-        {
+        } else if (option == "/v") {
           opt.userOpt.verbose = true;
-        }
-        else if (option == "/s")
-        {
+        } else if (option == "/s") {
           opt.userOpt.stress = true;
+        } else if (option == "/f") {
+          opt.userOpt.flatbuffer = true;
         }
```

Comment:
would be better to name based on what it is (ort format model) vs. the current internal storage mechanism (flatbuffer)